ci/build.ps1

```diff
@@ -135,11 +146,13 @@ $env:PATH = "C:\msys64\usr\bin;$env:PATH"
 & "C:\msys64\mingw$bits\bin\mingw32-make.exe" -C $(Convert-Path ..\src\nvim\testdir) VERBOSE=1
 
 if ($uploadToCodecov) {
-  C:\msys64\usr\bin\bash -lc "cd /c/projects/neovim; bash <(curl -s https://codecov.io/bash) -c || echo 'codecov upload failed.'"
+  bash -l /c/projects/neovim/ci/common/submit_coverage.sh oldtest ; exitAtEndIfFailed
 }
 
 # Build artifacts
 cpack -G ZIP -C RelWithDebInfo
 if ($env:APPVEYOR_REPO_TAG_NAME -ne $null) {
   cpack -G NSIS -C RelWithDebInfo
 }
+
+exit $exitAtEnd
```

Comment:
If code coverage fails to upload it should not fail the PR. Code coverage is a sanity-check. I don't want to see failures when it has problems.

Reply:
Failures there are not expected, and it's meant to catch issues with our setup - which would go silent otherwise (as while working on this PR).
But OTOH we'll also notice by dropped/missing coverage status otherwise, but then it is not directly visible which job(s) might have caused it.
I can revert this though of course, if you still want it.

Reply:
code coverage is low priority, it should never fail a PR because of network issues or whatever. So this should be reverted.

Reply:
ok.  otherwise good?

Reply:
yes, LGTM

Reply:
but the previous `|| echo 'codecov upload failed.'"` should be restored.